### PR TITLE
(v0.0.4) FEAT: Adding WikidataDescriptionField class

### LINKED ISSUE #11: Add support for a description field
Similar how SPARQL allows for label representation such as `?itemLabel`, consider adding a built-in field for `?itemDescription` that can be optionally applied

### PR DESCRIPTION
**Pull request by kennethsn**
closes #11

### CODE REVIEW

File: django_wikidata_api/fields.py [hunk 1]
```diff
@@ -98,32 +144,55 @@ def to_wikidata_group(self):
...
+
+        """
         # TODO: Merge similarities with entity list label
-        return "?{self.entity_name} rdfs:label ?{self.from_name} . ".format(self=self)
+        return "?{self.entity_name} {self.service_property} {self.from_name} . ".format(self=self)
 
-    def to_wikidata_group(self):
-        return "?{self.from_name}".format(self=self)
+
+class WikidataDescriptionField(WikidataLabelField):
```
Comment (kennethsn) [inline django_wikidata_api/fields.py:191]: This is the main contribution in this file:
Here I am creating a description field to add the `?itemDescription` argument to sparql queries like we do for  `?itemLabel`'s.

Other than this change, I'm adding docstrings and cleaning up some duplicated code

File: django_wikidata_api/models.py [hunk 2]
```diff
@@ -35,6 +35,7 @@ class Meta(object):
 
     main = WikidataEntityField(triples=[], required=True)
     label = WikidataLabelField(required=True)
+    description = WikidataDescriptionField()
```
Comment (kennethsn) [inline django_wikidata_api/models.py:38]: Here, I'm adding the description field to the Base model, so now by default all queries will pull in the description along with the label and qid

File: django_wikidata_api/tests/test_fields.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+from django_wikidata_api.fields import (
+    WikidataCharField,
+    WikidataDescriptionField,
+    WikidataField,
+    WikidataLabelField,
+)
+
+
+class WikidataFieldTests(TestCase):
```
Comment (kennethsn) [inline django_wikidata_api/tests/test_fields.py:18]: In this file, I'm adding tests for the new description field class and all it's parent classes

File: django_wikidata_api/tests/test_models.py [hunk 4]
```diff
@@ -16,20 +16,23 @@ class WikidataItemBaseTests(TestCase):
...
+        self.test_item_full = WikidataItemBase(label="Test Item", description="Some Test Description", main="Q123",
+                                               id=123, alt_labels=["Test"])
         self.mocked_query_response = {
             'results': {
                 'bindings': [
                     {
                         'main': {'type': 'uri', 'value': 'http://www.wikidata.org/entity/Q123'},
                         'mainLabel': {'xml:lang': 'en', 'type': 'literal', 'value': 'Test Item'},
                         'label': {'xml:lang': 'en', 'type': 'literal', 'value': 'Test Item'},
+                        'description': {'xml:lang': 'en', 'type': 'literal', 'value': 'Some Test Description'},
```
Comment (kennethsn) [inline django_wikidata_api/tests/test_models.py:28]: In this file, I'm making sure the full integration with Wikidata will successfully parse the descriptions in the Wikidata Query Service response format

File: django_wikidata_api/tests/test_models.py [hunk 5]
```diff
@@ -262,12 +284,12 @@ def test_build_query(self):
         self.assertNotIn("\t", output)
         # test values empty
         self.assertNotIn("VALUES", output)
-        self.assertIn("?main ?mainLabel (?mainLabel AS ?label) (GROUP_CONCAT(DISTINCT ?main_alt_label; SEPARATOR=\'|\')"
-                      " AS ?alt_labels)", output)
+        self.assertIn("?main ?mainLabel (?mainLabel AS ?label) ?mainDescription (?mainDescription AS ?description) "
```
Comment (kennethsn) [inline django_wikidata_api/tests/test_models.py:287]: This is how descriptions are structured in the sparql query

**Review by emulatingkat — APPROVED** [review]
Thank you for this work

**Pull request merged**